javascript/javascript2/week3/fastestPresserredo.js

```diff
@@ -0,0 +1,82 @@
+const timeInput = document.querySelector('#timeInputMarkup');
+const startGameButton = document.querySelector('#startGameMarkup')
+let timeInputValue;
+let setTimer;
+let secs;
+
+startGameButton.addEventListener('click', startGameFunc);
+document.getElementById('restartGameMarkup').addEventListener('click', startGameFunc)
+
+function startGameFunc() {
+    timeInputValue = timeInput.value;
+    clearContent();
+    if(timeInputValue === '') {
+        alert('Time is not specifed so default time set to 5seconds and starts the game')
+        timeInputValue = 5; // default time set to 5seconds if user doesent specifies time
+    }
+    secs = 1;
+    setTimer = setInterval(countTimer, 1000);
+    document.addEventListener('keypress', logKey);
+    setTimeout(delayTime, timeInputValue * 1000);
+}
+
+let countS = 0;
+let countL = 0;
+function logKey(e) {
+    const keyPressed = e.key.toLowerCase();
+    if(keyPressed === 's') {
+        countS += 1;
+        document.getElementById('countSMarkup').innerText = countS;
+    } else if(keyPressed === 'l') {
+        countL += 1;
+        document.getElementById('countLMarkup').innerText = countL;
+    }
+    console.log(keyPressed, countS, countL)
+}
+
+function delayTime() {
+    document.removeEventListener('keypress', logKey);
```

Comment:
It is recommended to use 'keydown' instead of 'keypress'. They have almost the same functionality, so you can(in this case) use them interchangeably.